DayPicker/DaySelectorDecorator.js

```diff
@@ -35,12 +35,17 @@ function generalizeSelected (selected, state) {
 	return selected.map(v => generalizeDay(v, state.firstDayOfWeek)).sort();
 }
 
-// Accepts a localized selected array and returns a "Sunday at index 0" array
+// Accepts a "Sunday at index 0" selected array or number and returns a
+// localized array or number.
 function localizeSelected (selected, state) {
-	if (state.firstDayOfWeek === 0 || !selected) {
+	if (typeof selected === 'undefined' || selected === null) {
```

Comment:
The original code had the intent to skip the "localizeDay" when the `firstDayOfWeek` is 0.
I don't see any reason for removing that condition.
How about
```suggestion
	if (state.firstDayOfWeek === 0 || selected == null) {
```

Reply:
Also, `selected == null` will return `true` when `selected` is `undefined` or `null`. You can use this.

Reply:
I will use it to revise the conditional statement.